xla/client/executable_build_options.h

```diff
@@ -236,6 +237,22 @@ class ExecutableBuildOptions {
 
   absl::StatusOr<ExecutableBuildOptionsProto> ToProto() const;
 
+  int process_index() const { return process_index_; }
```

Comment:
Could we add docstrings to these methods?

Reply:
What would you like to see in that docstring besides that it obviously returns the process index?

Reply:
OK fair, I see the concept of `process_index` already exists in PjRT.